lib/core/include/irods/client_connection.hpp

```diff
@@ -18,7 +18,9 @@ namespace irods::experimental
     /// authentication occurs.
     ///
     /// \since 4.2.9
-    struct defer_connection {} defer_connection;
+    const inline struct defer_connection
+    {
+    } defer_connection;
```

Comment:
This fix is part of #6673. I meant to pull that out into a separate commit for @epsilon-phase but forgot to do so.

I'll go ahead and do that today (along with any other small changes) and then we can rebase other PRs on top those.

Reply:
Sounds good